maintainers/maintainer-list.nix

```diff
@@ -4765,6 +4765,15 @@
     github = "seqizz";
     name = "Gurkan Gur";
   };
+  servalcatty = {
+    email = "servalcat@pm.me";
+    github = "servalcatty";
```

Comment:
@servalcatty as per brand new policy if you are including `github` please also include your `githubid`. Your `githubid` is [51969817](https://api.github.com/users/servalcatty).